worldedit-core/src/main/java/com/fastasyncworldedit/core/util/MathMan.java

```diff
@@ -200,6 +206,33 @@ public static int tripleSearchCoords(int x, int y, int z) {
                 + ((b4 & 0xFF) << 23));
     }
 
+    /**
+     * Obtain an integer representation of 3 ints, x, y and z. y is represented by the right-most 9 bits and can
+     * be within the range -256 to 255. x and z are represented by 11 bits each and can be within the range
+     * -1024 to 1024
+     *
+     * @param x x value
+     * @param y y value
+     * @param z z value
+     * @return integer representation of x y z
+     */
+    public static int tripleSearchCoordsExtendedY(int x, int y, int z) {
```

Comment:
Why aren't we using 10 bits for y here?

Reply:
Can't use the 32nd bit of an integer if we still want to use it as an index for arrays

Reply:
I see. Should be documented then I think

Reply:
> I see. Should be documented then I think

Agreed.